style-desktop.css

```diff
@@ -0,0 +1,316 @@
+    border: none;
+    color: #3a4a42;
+    background: #ebecf0;
+    border-radius: 4px;
+  }
+
+  .btn--frame {
+    padding: 0 11px;
+    font-size: 16px;
+    line-height: 16px;
+    background-color: #fff;
+    color: #36b37e;
+    width: auto;
+    height: 40px;
+    font-weight: bold;
+    border: none;
+    border-radius: 4px;
+  }
+
+  h1 {
+    font-size: 48px;
+    text-align: center;
+    line-height: 52px;
+    margin: 0;
+  }
+
+  .main--section {
+    display: flex;
+    width: 100%;
+    height: 1220px;
+    background: none;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: 70px;
+    background-image: url(./assets/back-img/Header-llustration-desktop@2x.png);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  .arrow--icons {
+    margin-top: 90px;
+    padding: 0;
+  }
+
+  .section--title {
+    margin-top: 160px;
+  }
+
+  .section--title h1,
+  .recent--work--title h1,
+  .about-title h1 {
+    margin: 0;
+    color: #172b4d;
+    font-family: 'Inter', sans-serif;
+    text-align: center;
+    letter-spacing: 0.37px;
+    font-size: 48px;
+    font-weight: 800;
+    line-height: 60px;
+  }
+
+  .paragraphe {
+    text-align: center;
+    justify-content: center;
+    font-size: 20px;
+    width: 920px;
+    line-height: 32px;
+    padding: 0;
+    margin: 0 954px;
+  }
+
+  .paragraphe p {
+    text-align: center;
+    justify-content: center;
+    padding: 12px;
+    margin: 24px;
+  }
+
+  .social--icons li {
+    padding-top: 58px;
+    margin-right: 20px;
+    width: 19px;
+    height: 20px;
+  }
+
+  .main--works--section {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 24px;
+    grid-auto-rows: 474px;
+    height: auto;
+    text-align: center;
+    padding: 0 750px;
+    padding-top: 24px;
+    margin-top: 90px;
+  }
+
+  .main--frames--section {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 24px;
+    grid-auto-rows: 470px;
+    text-align: center;
+    margin: 0;
+    padding: 20px 750px;
+    padding-top: 24px;
+  }
+
+  .card--section {
+    margin-right: 100px;
+    width: 100%;
+    height: 100%;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+  }
+
+  .intro--section {
+    text-align: center;
+    justify-content: center;
+    padding-bottom: 63px;
+    margin: 0;
+    width: 100%;
+  }
+
+  .about-title {
+    margin: 0;
+    padding-top: 192px;
+  }
+
+  .recent--work--section,
+  .about--me--section {
+    padding: 0;
+    margin: 0;
+  }
+
+  .about--me--section {
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    justify-content: center;
+    background: none;
+    background-image:
+      url(./assets/back-img/illustration-left-about-me-2-desktop@2x.png),
+      url(./assets/back-img/illustration-right-about\ me-1-desktop@2x.png);
+    background-position: left 27%, right 3%;
+    background-size: 11%;
+    background-repeat: no-repeat;
+  }
```

Comment:
- [ ] Kindly make the about-me section of your project look like that of Figma's template that you chose. I would like to recommend that you use the same CSS declarations for `fonts`, `margins`, `paddings`, and `colors` used for this section.
- [ ] I would like to recommend that you use `display: flex` to style this section. Using flexbox makes the page very responsive if you apply the right properties for the flexbox.

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144665314-28c57134-ec7b-4d8f-8e0a-5f4998ff94a0.png)

## Figma Template

![image](https://user-images.githubusercontent.com/77106962/144665440-55fa4250-1f36-44dd-a3ec-bf26389ef978.png)

### Kindly implement this change 👍